src/sw.prod.js

```diff
@@ -34,19 +36,19 @@ function fragmentFallbackStrategy({event, url}) {
     });
 }
 workbox.routing.registerRoute(
-  new RegExp('\/(.+)?fragment=true'),
+  new RegExp('/(.+)?fragment=true'),
```

Comment:
These changes are completely out of scope for this PR. Please move to a new PR so that we can test separately.

Reply:
I get that, but there are linting errors if I leave them.

I can fix the existing errors in a separate PR if you'd prefer.

Reply:
Which linter are you using? npm run lint is not reporting anything on my machine nor on project's travis. Maybe we have different configs?

Reply:
Hmm...

I got these error reports by calling `npm run lint`.

This is using .eslintrc.yaml, right?

The linting errors looks correct. Like I say, I can move the fixes to a new PR if that's easier.

Reply:
What are the linter errors? Something from `"avoidEscape": true`? I'm not seeing them fire either which is odd.

Reply:
The problem here is this (which makes sense): https://eslint.org/docs/rules/no-useless-escape

I wonder why you're (both) not seeing these errors.

Did you get the other (undefined `path`) error reported?

Do you have the _.eslintignore_ and _.eslintrtc.yaml_ files from master?

I'm using eslint v4.16.0 via `npm run lint`.